.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,11 +1,11 @@
 <!--- Provide a general summary of your changes in the Title above -->
 
-## Checklist for submitting an `awesome` plugin, theme, or resource:
-<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
-<!--- More detailed instructions are in the CONTRIBUTING.md document - please read it if you have a question!-->
+## Checklist for submitting a resource to `awesome-hyper`:
+<!--- Go over all the following points, and put an `x` in all the boxes that have been completed. -->
+<!--- More detailed instructions are in the project's CONTRIBUTING.md - please read it if you have a question!-->
 <!--- If you're still unsure about any of these, don't hesitate to ask. We're here to help! -->
-- [ ] The title for my package or theme uses its npm title (`hyper-plugin-example`)
-- [ ] The link for my package or theme uses the npmjs.com link (https://www.npmjs.com/package/hyper-plugin-example)
-- [ ] There is a visual representation of what my plugin or theme does in the repo. (Plugin: screenshot or gif || Theme: screenshot of the theme applied to Hyper)
-- [ ] Put your awesome item at the **BOTTOM** of the correct (plugin, theme, or resource) list.
-- [ ] **VERY IMPORTANT:** I've written a short (one sentence) description for my package or theme as the description of the awesome plugin, theme, or resource in the README.md file I'm submitting in the PR.
+- [ ] **Title:** The title for the addition uses its npm title (`hyper-example`) or the resource's title.
+- [ ] **Link:** The link for the addition uses the npmjs.com link (`https://www.npmjs.com/package/hyper-example`) or the resource's full URI.
```

Comment:
this is always hitting me, no one ever in the history of this repo update this specific point correctly. so in order to check out package, you have to go to the diff and manually copy the url =(